packages/ERTP/package.json

```diff
@@ -49,9 +48,10 @@
   },
   "devDependencies": {
     "@agoric/bundle-source": "^1.1.5",
+    "@agoric/install-ses": "^0.1.0",
     "@agoric/swingset-vat": "^0.5.2",
     "esm": "^3.2.25",
-    "tap-spec": "^5.0.0",
+    "tap": "^14.10.5",
     "tape": "^4.11.0",
```

Comment:
Should we get rid of `tape` and `tape-promise`?

Reply:
I think I'm confused about which is being used.

Reply:
At this stage we need to keep them both. We're using `tap` as the test runner (`tap 'test/**/test*.js'`): the driver's-seat program which finds all your other test programs, runs them, collects their output, and decides whether they succeeded or not (and probably displays a reformatted/filtered version of their output). Nearly all of our test programs import `tape` and use it for all their assertions. So we need both for now.

Once SES tolerates changes to the `Error` object (which a lot of testing tools modify, to extract stack traces and such), we should be able to experiment with other test runners and other assertion libraries. So I'm thinking of `tap` as a good-enough-for-now step. I'd like to conduct a survey of testing tools and see if we'd like to move to something else in the future.

Reply:
Ok sounds good!